xml/containers_basics.xml

```diff
@@ -1,78 +1,190 @@
+    </para>
+    <para>
+      While it is possible to create an image from scratch, using an existing base image is more practical in most situations. Base image has no parent, meaning it is not based on another image.
+    </para>
+    <para>
+      You can use a base image as it is for running containers, but the main purpose of base images is to serve as foundations for creating custom images that can run containers with specific applications, servers, services, and so on.
+    </para>
+    <para>
+      Both base and custom images are usually available through a repository of
+      images called registry. unless a registry is explicitly specified,
+      &podman; pulls images from the Docker Hub registry.
+    </para>
+    <para>
+      After you pull a base image, you can use it as a starting point for building a custom image. For example, you can use the SLE base image to generate an image that is configured to run a specific application. To build a custom image, you need to create a special file called Dockerfile, containing building instructions. For example, a Dockerfile can contain instructions to update the system software, install the desired application, open specific network ports, run commands, etc.
+    </para>
+    <para>
+      Note that you can build images not only from base images, but also on top of custom images. So you can have an image consisting of multiple layers:
+    </para>
+    <para>
+      Base image -> Custom image 1 -> Custom image 2 -> Custom image 3 (
+      <!--TODO Replace with figure -->
+    </para>
+    <sect2 xml:id="sec-container-example">
+      <title>Practical Example</title>
+      <para>
+        The following procedure shows how to build a custom &docker; image that
+        can be used to run a container with a simple PHP application called
+        <literal>example</literal>
+        served using the built-in PHP development
+        server.
+      </para>
+      <procedure>
+        <title>Building an image and running container</title>
+        <step>
+          <para>
+            Install &podman;:
+          </para>
+<screen>sudo zypper in podman</screen>
+        </step>
+        <step>
+          <para>
+            Switch to the PHP project's directory and create a file named
+            <filename>Dockerfile</filename>
+            :
+          </para>
+<screen>cd example
+touch Dockerfile</screen>
+        </step>
+        <step>
+          <para>
+          </para>
+        </step>
+        <step>
+          <para>
+            Open the <filename>Dockerfile</filename> file for editing, and add the following:
+          </para>
+<screen>
+FROM php:7.4-cli
+COPY . /usr/src/example
+WORKDIR /usr/src/example
+EXPOSE 8000
+CMD [ "php", "-S", "0.0.0.0:8000" ]</screen>
+        </step>
+        <step>
+          <para>
+            Build a container image:
+          </para>
+<screen>sudo podman build -t example .</screen>
```

Comment:
As it's about podman now, this can make use of the rootless mode now. I'm not sure which steps are needed for setup though, @sysrich might be able to tell. Is it still necessary to set up `subuid`/`subgid` beforehand?

Reply:
Rootless Podman should definitely be covered, but in the dedicated Podman chapter.

Reply:
It's as easy as just omitting the `sudo`, so should be the default if possible